ui/perfherder/constants.js

```diff
@@ -32,7 +32,7 @@ export const selectorCardText = {
 };
 
 export const compareTableText = {
-  retriggerButtonTitle: 'Retrigger 5 times',
+  retriggerButtonTitle: 'Retrigger jobs',
```

Comment:
`'Retrigger jobs'` is used in two different places. Suggestion: use a constant variable.